src/app/App.ts

```diff
@@ -4,14 +4,14 @@ import { WebChessView } from './view/web/WebChessView';
 import { ChessEngine } from './model/ChessEngine';
 import { ChessModel } from './model/ChessModel';
 import { InMemoryViewEventBus } from './view/events/InMemoryViewEventBus';
-import { ChessBoard } from './model/ChessBoard';
+import { Chessboard } from './model/Chessboard';
 import { SquareWithPiece } from './model/Types';
 
 export const App = (): void => {
   const viewEventBus = new InMemoryViewEventBus();
   const chessBoardView: ChessBoardView = new WebChessView(viewEventBus);
   const boardWithPieces: SquareWithPiece = {};
-  const chessBoard: ChessBoard = new ChessBoard(boardWithPieces);
+  const chessBoard: Chessboard = new Chessboard(boardWithPieces);
```

Comment:
Czy tej zmiany dotyczył PR? Czy to dodatkowa zmiana? Jesli dodatkowa to proponuję do osobnego PR

Reply:
tak, ale już bez sensu to przerzucać, bo za dużo zmian mam u siebie